inst/tests/tests.Rraw

```diff
@@ -18532,3 +18532,15 @@ test(2260.15, inherits(tryCatch(messagef("x", class="test_message"), condition=i
 test(2260.16, inherits(tryCatch(messagef("x", class="test_message"), condition=identity), "message"))
 test(2260.17, inherits(tryCatch(packageStartupMessagef("x", class="test_psm"), condition=identity), "test_psm"))
 test(2260.18, inherits(tryCatch(packageStartupMessagef("x", class="test_psm"), condition=identity), "packageStartupMessage"))
+
+test(2261.01, setNumericRounding(0L), 0L)
+test(2261.02, getNumericRounding(), 0L)
+test(2261.03, setNumericRounding(1L), 0L)
```

Comment:
it looks like some of these tests are redundant, can you explain why we need 11 tests?

Reply:
yeah it was in the commit message, i just had these for a sanity check, but can certainly cut it down

Reply:
Let's try and make it more minimal, thanks. If we think there's added value of tests that might appear redundant, a good comment will help future readers.